native/cocos/3d/assets/Mesh.cpp

```diff
@@ -1071,9 +1091,148 @@ void Mesh::accessAttribute(index_t primitiveIndex, const char *attributeName, co
     }
 }
 
-/* static */
+void Mesh::updateVertexFormat() {
```

Comment:
It is better to rename `convertVertexData`  and divide it into some small functions, such as:
- convertRGB32F_2_RGB16F
- ...